services/people.rb

```diff
@@ -0,0 +1,41 @@
+require 'lib/ir_pagerduty'
+
+class People
+  def fetch(path, params={})
+    pd = IRPagerduty.new
+    pd.fetch_json(path, params)
+  end
+
+  def fetch_users(params)
+    fetch("users", params)['users']
+  end
+
+  def fetch_schedules(params)
+    fetch("schedules", params)['schedules']
+  end
+
+  def fetch_schedules_users(schedule_id, params)
+    fetch("schedules/#{schedule_id}/users", params)['users']
+  end
+
+  def fetch_users_contact_methods(user_id)
+    fetch("users/#{user_id}/contact_methods")['contact_methods']
+  end
+
+  def fetch_todays_shedules_users(sid)
+    today = Date.today
+    users = fetch_schedules_users(sid, {
+      :since => today.strftime('%FT%TZ'),
+      :until => (today + 1).strftime('%FT%TZ')
+    })
```

Comment:
I think we can add `'includes[] => 'contact_methods' here to combine both these api calls into one though.

(Shame the PagerDuty::Full doesn't expose this)